scripts/linters/find_unused_deps.rb

```diff
@@ -13,7 +13,12 @@ def get_pattern(crate_raw)
 # Special cases to suppress false positives.
 def excluded?(crates, crate)
   # `quickcheck` is required implicitly by `quickcheck_macros`
-  crate == 'quickcheck' && crates.include?('quickcheck_macros')
+  if crate == 'quickcheck' && crates.include?('quickcheck_macros')
+      return true
+  end
+  # we are using a trick to enable a `test-utils` feature in the `f3-gpbft` which
+  # triggers a false positive.
+  crate == 'filecoin-f3-gpbft'
```

Comment:
nit: why not `crate == 'quickcheck' && crates.include?('quickcheck_macros') or crate == 'filecoin-f3-gpbft'`?

Reply:
I think it's much more readable like it is right now. If we need to introduce more deps - perhaps we have to revisit the approach. Having a long chain of `&&` and `or` isn't much IMHO. No strong opinion.

Reply:
Might I suggest:
```ruby
  [
    # `quickcheck` is required implicitly by `quickcheck_macros`
    crate == 'quickcheck' && crates.include?('quickcheck_macros'),
    # we are using a trick to enable a `test-utils` feature in the `f3-gpbft` which
    # triggers a false positive.
    crate == 'filecoin-f3-gpbft'
  ].any?
```
?

Reply:
I like this